src/main/java/seedu/mtracker/console/AddForexParser.java

```diff
@@ -33,17 +33,26 @@ public void addForexRemarksToParameter() {
     }
 
     public void addForexEntryToParameter() {
-        String entryPrice = getForexEntryFromUser();
+        String entryPrice;
+        do {
+            entryPrice = getForexEntryFromUser();
+        } while (!isValidPrice(entryPrice));
         parameters.add(entryPrice);

```

Comment:
Nice, I think we shld do this way for the other parameters, will create new issue from this